include/fem-dem/fluid_dynamics_vans.h

```diff
@@ -109,6 +109,40 @@ class FluidDynamicsVANS : public FluidDynamicsMatrixBased<dim>
   virtual void
   setup_dofs() override;
 
+  /**
+   * @brief Set the initial condition. The version in this class does not output
+   * the solution, since the void fraction is not initialized yet.
+   *
+   * @param initial_condition_type Type of method  use to impose initial condition.
+   *
+   * @param restart Indicator if the simulation is being restarted or not.
+   *
+   **/
+
+  void
+  set_initial_condition(
+    Parameters::FluidDynamicsInitialConditionType initial_condition_type,
```

Comment:
```suggestion
    const Parameters::FluidDynamicsInitialConditionType initial_condition_type,
```

Reply:
I would modify the base class to maintain consistency. But these two parameters should be const.